src/main/java/org/assertj/core/internal/Objects.java

```diff
@@ -721,6 +722,7 @@ static boolean propertyOrFieldValuesAreEqual(Object actualFieldValue, Object oth
    * @param actual                     the given object.
    * @param propertiesOrFieldsToIgnore the fields to ignore in comparison.
    * @throws AssertionError if actual is {@code null}.
+   * @throws AssertionError if any of the properties or fields to ignore doesn't exist.
```

Comment:
this should be added to the `AbstractObjectAssert#hasNoNullFieldsOrPropertiesExcept` javadoc with a note saying it is a breaking change introduced in 3.20.0